content/docs/README.md

```diff
@@ -1,23 +1,19 @@
 ---
 title: cert-manager
-description: cert-manager documentation homepage
+description: |
+    cert-manager creates signed TLS / SSL certificates for workloads in your Kubernetes or OpenShift cluster
+    and renews the certificates before they expire.
 ---
 
-cert-manager adds certificates and certificate issuers as resource types in
-Kubernetes clusters, and simplifies the process of obtaining, renewing and
-using those certificates.
+cert-manager creates signed TLS / SSL certificates for workloads in your Kubernetes or OpenShift cluster
+and renews the certificates before they expire.
+Typically, the private key and signed certificate are stored in Kubernetes secrets
+and used by applications or ingress controllers.
 
-It can issue certificates from a variety of supported sources, including
-[Let's Encrypt](https://letsencrypt.org), [HashiCorp Vault](https://www.vaultproject.io),
-and [Venafi](https://www.venafi.com/) as well as private PKI.
+cert-manager can obtain signed certificates from a variety of certificate authorities, including:
```

Comment:
```suggestion
cert-manager can obtain certificates from a variety of certificate authorities, including:
```
Not sure that we have to mention that these certificates are signed, I think all certificates are signed.